src/cmd_all/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 
 [features]
 static-link = ["workspace-static-link"]
+sync_point = ["sync-point/sync_point"]
```

Comment:
Don't have to add a feature here. A top-level `--feature sync-point/sync_point` is enough.

Reply:
Right. Thanks.